README.md

```diff
@@ -1,113 +1,55 @@
-    },
-    "build": {
-        "number": 22,
-        "status": "success",
-        "started_at": 1421029603,
-        "finished_at": 1421029813,
-        "message": "Update the Readme",
-        "author": "johnsmith",
-        "author_email": "john.smith@gmail.com",
-        "event": "push",
-        "branch": "master",
-        "commit": "436b7a6e2abaddfd35740527353e78a227ddcb2c",
-        "ref": "refs/heads/master"
-    },
-    "workspace": {
-        "root": "/drone/src",
-        "path": "/drone/src/github.com/drone/drone"
-    },
-    "vargs": {
-        "from": "noreply@foo.com",
-        "host": "smtp.mailgun.org",
-        "port": 587,
-        "username": "",
-        "password": "",
-        "recipients": [
-            "octocat@github.com"
-        ]
-    }
-}
-EOF
-```
+Build the docker image with the following commands:
 
-## Docker
+```
+CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -tags netgo
+docker build -t plugins/email .
+```
 
-Build the container using `make`:
+Please note incorrectly building the image for the correct x64 linux and with GCO disabled will result in an error when running the Docker image:
 
 ```
-make deps docker
+docker: Error response from daemon: Container command
+'/bin/drone-email' not found or does not exist..
 ```
 
 ### Example
+Execute from the working directory:
 
 ```sh
-docker run -i plugins/drone-email <<EOF
-{
-    "repo": {
-        "clone_url": "git://github.com/drone/drone",
-        "owner": "drone",
-        "name": "drone",
-        "full_name": "drone/drone"
-    },
-    "system": {
-        "link_url": "https://beta.drone.io"
-    },
-    "build": {
-        "number": 22,
-        "status": "success",
-        "started_at": 1421029603,
-        "finished_at": 1421029813,
-        "message": "Update the Readme",
-        "author": "johnsmith",
-        "author_email": "john.smith@gmail.com"
-        "event": "push",
-        "branch": "master",
-        "commit": "436b7a6e2abaddfd35740527353e78a227ddcb2c",
-        "ref": "refs/heads/master"
-    },
-    "workspace": {
-        "root": "/drone/src",
-        "path": "/drone/src/github.com/drone/drone"
-    },
-    "vargs": {
-        "from": "noreply@foo.com",
-        "host": "smtp.mailgun.org",
-        "port": 587,
-        "username": "",
-        "password": "",
-        "recipients": [
-            "octocat@github.com"
-        ]
-    }
-}
-EOF
+docker run --rm \
+  -e PLUGIN_FROM=drone@test.test \
+  -e PLUGIN_HOST=smtp.test.test \
+  -e PLUGIN_USERNAME=drone \
+  -e PLUGIN_PASSWORD=test
+  -e DRONE_REPO_OWNER=octocat \
+  -e DRONE_REPO_NAME=hello-world \
+  -e DRONE_COMMIT_SHA=7fd1a60b01f91b314f59955a4e4d4e80d8edf11d \
+  -e DRONE_COMMIT_BRANCH=master \
+  -e DRONE_COMMIT_AUTHOR=octocat \
+  -e DRONE_COMMIT_AUTHOR_EMAIL=octocat@test.test \
+  -e DRONE_BUILD_NUMBER=1 \
+  -e DRONE_BUILD_STATUS=success \
+  -e DRONE_BUILD_LINK=http://github.com/octocat/hello-world \
+  -e DRONE_COMMIT_MESSAGE="Hello world!" \
+  plugins/email
```

Comment:
For all other plugins we also set the volume and working directory, even if it's not really important for this plugin.